.github/workflows/release-drafter.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         id: md_writer
         if: github.event_name != 'pull_request' && startsWith(github.ref, 'refs/tags/') && success()
         run: |
-          printf '%b\n' "---\nlayout: default\n---\n\n${{ steps.release_drafter.outputs.body }} " > ./docs/_releases/${{ steps.release_drafter.outputs.tag_name }}.md
+          printf '%b\n' "---\nlayout: default\ndate: $(date +'%d-%m-%Y')\n---\n\n${{ steps.release_drafter.outputs.body }} " > ./docs/_releases/${{ steps.release_drafter.outputs.tag_name }}.md
```

Comment:
```suggestion
          printf '%b\n' "---\nlayout: default\ndate: $(date +'%m-%d-%Y')\n---\n\n${{ steps.release_drafter.outputs.body }} " > ./docs/_releases/${{ steps.release_drafter.outputs.tag_name }}.md
```

Reply:
We should change the date format in previous release file as well (currently it is "%d-%m-%Y")